tests/test-data/keys/ecdsa_prime256v1.pem

```diff
@@ -0,0 +1,5 @@
+-----BEGIN EC PRIVATE KEY-----
+MHcCAQEEIKE0vihID+ffiAB2O8DW7d9kTRcGHA7a7bsr0lu2NPaKoAoGCCqGSM49
+AwEHoUQDQgAEc563pICgt7T27Hu1nnJ1Qis6jwIEx0ixBihKkeJRurqz/2EZtpmi
+o263hkYfxpgr2T0dvqu/qfSS5Jbc14Iw3A==
+-----END EC PRIVATE KEY-----
```

Comment:
The issue was with `openssl dgst`, not the key generation, but this prompted me to go back and look and sure enough it takes a `keyform` that I missed before. `openssl genrsa` doesn't seem to allow selecting the output format, but that's what `rm` is for...